sdk/tests/deqp/functional/gles3/es3fNegativeBufferApiTests.js

```diff
@@ -564,22 +564,23 @@ goog.scope(function() {
                 gl.checkFramebufferStatus(gl.FRAMEBUFFER);
                 this.expectError(gl.NO_ERROR);
 
-                bufferedLogToConsole('gl.INVALID_ENUM is generated if one of the values in bufs is not an accepted value.');
-                gl.drawBuffers(values.splice(2, 2));
-                this.expectError(gl.INVALID_ENUM);
+                bufferedLogToConsole('gl.INVALID_OPERATION is generated if the GL is bound to framebuffer object and one of the values in bufs is not an accepted value.');
+                gl.drawBuffers(values.slice(0, 2));
+                this.expectError(gl.INVALID_OPERATION);
```

Comment:
GLES3 man page says that there should be an INVALID_ENUM (see https://www.khronos.org/opengles/sdk/docs/man3/html/glDrawBuffers.xhtml), but GLES 3.0.4 spec says that there should be an INVALID_OPERATION. see https://www.khronos.org/registry/gles/specs/3.0/es_spec_3.0.4.pdf#nameddest=section-4.2.1 (p185 - p186)